index.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import {
+  DeviceEventEmitter,
+  NativeModules
+} from 'react-native';
+
+const {MobileDeviceManager} = NativeModules;
+
+MobileDeviceManager.addListener = (callback) => {
```

Comment:
Hrmm, don't love modifying the native module. Not sure if this will work, but can we mix it into another object?

```js
export default {
  ...MobileDeviceManager,
  addListener (callback) { ... }
}
```